### PR TITLE
Basic Docker and commands howto

### LINKED ISSUE #18: Add short howto describing how to launch the app with Docker

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #18

**How to prepare for test**:
- Have Docker

### How to test:
- [x] Follow the new instructions in the README file

### Expected outcome:
- [x] The endpoints should be available

### Review:
- [ ] Code approved by
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
`docker.compose up`:

image

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
-# Schug 
+# Schug
 
-Schug :stew: is a service that gather data about genes, transcripts and exons from multiple sources and merge the 
+Schug :stew: is a service that gather data about genes, transcripts and exons from multiple sources and merge the
 information. There is a [REST API][rest-api] with relevant endpoints.
 
+## Test the app using Docker
+
+You can test Schug on your local computer using Docker. Make sure you have [Docker][docker] installed and type the following command in a terminal window: `docker-compose up`
```
Comment (henrikstranneheim) [inline README.md:8]: Don’t you have to be in the root for or point to the docker compose file?

**Review by henrikstranneheim — APPROVED** [review]
Nice!👍

File: README.md (hunk 1 above)
Comment (northwestwitch) [reply README.md:8]: Ah yes, I'll add it!

Comment (northwestwitch) [reply README.md:8]: I guess you also pull the image from our Docker Hub, but we don't have a prod image yet, so I guess it's better to write to download it and go to the root

**Pull request merged**